core/server/common/src/main/java/alluxio/RpcUtils.java

```diff
@@ -37,10 +37,11 @@
     try {
       return callable.call();
     } catch (AlluxioException e) {
-      logger.debug("Exit (Error): {}", callable, e);
+      logger.warn("Exception calling: {}, Error={}", callable, e.getMessage());
```

Comment:
I actually don't think we should log warn here, since we use exceptions for control flow, it will become very noisy. Unfortunately this will break the logging convention. I think these two lines should be consolidated into:

`logger.debug("{}, Error={}", callable, e.getMessage());`